src/components/Offer/Offer.tsx

```diff
@@ -179,10 +179,14 @@ export const Offer: React.FC<OfferProps> = ({
         <SellerPreviewInfo image={profile} name={name} location={location} />
         <DescriptionOffer text={offerDescription} />
         <div className='buttonContainer'>
-          <Button isPrimary={isLearnEnabled} label={detailButtonText} />
+          <Button
```

Comment:
Can we change the styles using styled-components instead of using classNames?

Reply:
Truly!